contracts/modules/ProtocolSettings.sol

```diff
@@ -311,43 +312,62 @@ contract ProtocolSettings is State, ProtocolTokenUser, ProtocolSettingsEvents {
 	/**
 	 * @notice The feesController calls this function to withdraw fees
 	 * from three sources: lending, trading and borrowing.
+	 * The fees will be converted to wRBTC
 	 *
-	 * @param token The address of the token instance.
+	 * @param tokens The array address of the token instance.
 	 * @param receiver The address of the withdrawal recipient.
 	 *
-	 * @return The withdrawn amount.
+	 * @return The withdrawn total amount in wRBTC
 	 * */
-	function withdrawFees(address token, address receiver) external returns (uint256) {
+	function withdrawFees(address[] calldata tokens, address receiver) external returns (uint256) {
 		require(msg.sender == feesController, "unauthorized");
+		uint256 totalWRBTCWithdrawn;
 
-		uint256 lendingBalance = lendingFeeTokensHeld[token];
-		if (lendingBalance > 0) {
-			lendingFeeTokensHeld[token] = 0;
-			lendingFeeTokensPaid[token] = lendingFeeTokensPaid[token].add(lendingBalance);
-		}
+		for (uint256 i = 0; i < tokens.length; i++) {
+			uint256 lendingBalance = lendingFeeTokensHeld[tokens[i]];
+			if (lendingBalance > 0) {
+				lendingFeeTokensHeld[tokens[i]] = 0;
+				lendingFeeTokensPaid[tokens[i]] = lendingFeeTokensPaid[tokens[i]].add(lendingBalance);
+			}
 
-		uint256 tradingBalance = tradingFeeTokensHeld[token];
-		if (tradingBalance > 0) {
-			tradingFeeTokensHeld[token] = 0;
-			tradingFeeTokensPaid[token] = tradingFeeTokensPaid[token].add(tradingBalance);
-		}
+			uint256 tradingBalance = tradingFeeTokensHeld[tokens[i]];
+			if (tradingBalance > 0) {
+				tradingFeeTokensHeld[tokens[i]] = 0;
+				tradingFeeTokensPaid[tokens[i]] = tradingFeeTokensPaid[tokens[i]].add(tradingBalance);
+			}
```

Comment:
i believe there extra spaces here which is weird as they are not regular, from time to time
can it be that the formatter hangs and you just cancel formatting periodically?
please check your prettier/IDE formating settings
Franklin might help with this if you can't resolve the issue

Reply:
Yeah, apparently i was set my tab size value to 4 instead of 2 as the default